x-pack/plugins/ml/public/application/routing/routes/data_frame_analytics/models_list.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
You can remove one of these comments.

Reply:
removed it in https://github.com/elastic/kibana/pull/74978/commits/e8a46b3afaf4847bdefe7cdff1f0bb24d6c164e2, my IDE collapses these comments and probably the eslint fix task has accidentally added an extra one when the file was invalid